File: src/app/models/UI/IGColumn.ts
import {IView} from './IView';
import {IViewGroup} from './IViewGroup';

export class IGColumn extends IViewGroup {
  span: number;
  controls: IView[];


  constructor(span: number, controls: IView[]) {
    super();
    this.span = span;
    this.controls = controls;
  }
}

File: src/main/java/com/webstudio/connectionhub/models/UI/IGColumn.java
package com.webstudio.connectionhub.models.UI;

public class IGColumn {
    int span;

    IView[] controls;


    public IGColumn() {
    }


    public int getSpan() {
        return span;
    }

    public void setSpan(int span) {
        this.span = span;
    }

    public IView[] getControls() {
        return controls;
    }

    public void setControls(IView[] controls) {
        this.controls = controls;
    }
}
